src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -518,21 +517,28 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
-  const renderBannerPhotoInput = () => (
-    <div className="input-container">
-      <h4 className="italic input-header">Upload a banner photo</h4>
-      <ImageInput
-        disabled={disable}
-        name={"bannerImageFile"}
-        image={values.bannerImageFile}
-        remoteUrlInputName={"bannerImageUrl"}
-        remoteImageUrl={values.bannerImageUrl}
-        register={register}
-        setValue={setValue}
-        error={errors.bannerImageFile || errors.bannerImageUrl}
-      />
-    </div>
-  );
+  const renderBannerPhotoInput = () => {
```

Comment:
as we're not going to have "banner photo" in UI, let's rename all its mentions to "highlight image"

Reply:
renamed to highlight in b396ea2